plutus-core/plutus-core/src/PlutusCore/Evaluation/Machine/ExBudgeting.hs

```diff
@@ -200,28 +200,28 @@ runTwoArgumentModel
     (ModelTwoArgumentsConstantCost c) _ _ = c
 runTwoArgumentModel
     (ModelTwoArgumentsAddedSizes (ModelAddedSizes intercept slope)) (ExMemory size1) (ExMemory size2) =
-        ceiling $ (fromInteger (size1 + size2)) * slope + intercept -- TODO is this even correct? If not, adjust the other implementations too.
+        ceiling $ (1000 *) $ (fromInteger (size1 + size2)) * slope + intercept -- TODO is this even correct? If not, adjust the other implementations too.
```

Comment:
All this multiplication by 1000 is because most of the slopes inside the ceiling are very small and you'd have to have enormous numbers to change the cost.  This makes the costs a lot more fine-grained, which is a lot better for the overall modelling process.  These factors may go away at some later point.

Reply:
I propose to make it a named constant then.

Reply:
Yes, I don't know why I didn't do that.

Reply:
Maybe eventually, once I've worked out what we want to do in the final cost model.